src/branding/default/branding.desc

```diff
@@ -106,6 +106,11 @@ strings:
 
 # These images are loaded from the branding module directory.
 #
+# productBanner is an optional image, which if present, will be shown
+#       on the welcome page of the application, above the welcome text.
+#       It is intended to have a width much greater than height.
+#       It is displayed at 64px height (also on HiDPI).
```

Comment:
The code says `64 + 2 * defaultFontHeight` which is probably 88 in total?

Reply:
your right - that exposed a think-o - i intended (2 * defaultFontHeight) as padding - my image actually is 64 px - i should have surrounded it with explicit "spacer" widgets